locale/af/alert.json

```diff
@@ -0,0 +1,54 @@
+{
+  "title": "title",
+  "intro": "intro",
+  "note.title": "note.title",
+  "note.items": ["note.item", "note.item", "note.item"],
+  "sections": [
+    {
+      "title": "sections.title",
+      "paragraphs": [
+        {
+          "image.key": "sections.paragraphs.image.key",
+          "image.alt": "sections.paragraphs.image.alt",
+          "text": "sections.paragraphs.text"
+        }
+      ],
+      "links": [
+        {
+          "text": "sections.links.text",
+          "url": "sections.links.text"
+        }
+      ]
+    },
+    {
+      "title": "sections.title",
+      "paragraphs": [
+        {
+          "image.key": "sections.paragraphs.image.key",
+          "image.alt": "sections.paragraphs.image.alt",
+          "text": "sections.paragraphs.text"
+        },
+        {
+          "image.key": "sections.paragraphs.image.key",
+          "image.alt": "sections.paragraphs.image.alt",
+          "text": "sections.paragraphs.text"
+        }
+      ],
+      "links": [
+        {
+          "text": "sections.links.text",
+          "url": "sections.links.url"
+        }
+      ]
+    }
+  ],
+  "inlineAlertHeading": "inlineAlertHeading",
+  "inlineParagraph": "inlineParagraph",
+  "infoAlertParagraph": "infoAlertParagraph",
+  "infoHeader": "infoHeader",
```

Comment:
This is not a header, but a heading or maybe a label. So this as well as `errorHeader` need adjusting. Also the naming is not logical. Some of these have the 'alert' word in their names and some don't and it doesn't seem to have any logic. Please make them uniform in that sense as well.

Reply:
Changed to Heading.

Reply:
The naming policy of these is still not clear to me though. Please fix that as well.

Reply:
fixed to "example.inlineAlertInfoHeading":